HOMEapp/README.md

```diff
@@ -181,10 +181,10 @@ Deployment
 
 -   Specify who can use the application, as follows:
 
--  Accounts in any organizational directory (Any Azure Ad directory --Multitenant) and personal Microsoft accounts.
+-   Accounts in any organizational directory (Any Azure Ad directory --Multitenant) and personal Microsoft accounts.
 
--   Under Redirect URI select Public client/native (mobile & desktop).Then enter the destination URI that will be accepted when returning authentication responses (tokens)
-    after successfully authenticating users. 
+-   Under Authentication, click on Add a platform and select Web.Then enter the redirect URI that will be accepted when returning authentication responses (tokens)
+    after successfully authenticating users. Make sure the redirect URI follows the following format, https://%appDomain%.azurewebsites.net/End. Redirect URI can be set post     deployment also, if the appDomain is not available yet.
 
```

Comment:
Under Authentication, click on Add a platform and select Web. Then enter the redirect URI that will be accepted when returning authentication responses (tokens).
              - After successfully authenticating users, make sure the redirect URI follows the following forma:, 
                 https://%appDomain%.azurewebsites.net/End. 
              - Note, redirect URI can be set post deployment also, if the appDomain is not available yet.

Reply:
Updated